mac/Keyman4Mac/Keyman4Mac/Frameworks/KeymanEngine4Mac.framework/Headers

```diff
@@ -0,0 +1 @@
+Versions/Current/Headers
```

Comment:
Upon further study of the code, both KMCompStore and KMCompGroup have description methods that already display everything we want to see (and then some), so we can just change the loop variable to be a NSObject and we're all set.